lib/rich-description.coffee

```diff
@@ -0,0 +1,23 @@
+marked = require 'marked'
+
+renderer = new marked.Renderer()
+renderer.code = ->
+  ''
+renderer.blockquote = ->
+  ''
+renderer.heading = ->
+  ''
+renderer.html = ->
+  ''
+renderer.image = ->
+  ''
+renderer.list = ->
+  ''
```

Comment:
From a purely aesthetic viewpoint, it might look a bit more tidy if all of these were collapsed to single lines;
~~~ coffee
renderer.code = -> ''
renderer.blockquote = -> ''
renderer.heading = -> ''
renderer.html = -> ''
renderer.image = -> ''
renderer.list = -> ''
~~~